frontend/app/src/pages/room/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Page } from "@/components"
+import { ROUTE } from "@/constants/route";
+import { Collaborator, InfoBar } from "@/features/room"
+import { VStack } from "@chakra-ui/react"
+import { useParams, Navigate } from "react-router-dom";
+
+function RoomPage() {
+
+    const { roomId } = useParams();
+
+    console.log(roomId)
+
+    return (
+        <Page display="grid" placeItems="center">
+            <VStack marginBottom="4" flexDirection="column" align="left" height="full" width="full">
+                <InfoBar difficulty={"Easy"} topic={"Data Structures & Algorithms"} />
+                // TODO: Add Error message if room id not passed as prop
```

Comment:
this comment will be rendered on the page - need to surround comment with {/* ... */}

Reply:
Fixed in [cf9f6e5](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/18/commits/cf9f6e5b1f2962a58df7e19a78f93e2211be1191)